frontend/src/js/controllers/changePwdCtrl.js

```diff
@@ -53,8 +53,8 @@
                         var status = response.status;
                         var response = response.data;
                         vm.user.error = false;
-                        console.log("PASSWORD CHANGED SUCCESSFULLY");
                         console.log(response);
```

Comment:
Great work :+1:
@anweshknayak: remove this console too. Then i'll merge this PR.
LGTM.